Extension/src/LanguageServer/settings.ts

```diff
@@ -162,7 +162,7 @@ export class CppSettings extends Settings {
         return path;
     }
 
-    public get maxConcurrentThreads(): number | undefined | null { return super.Section.get<number | null>("maxConcurrentThreads"); }    public get maxMemory(): number | undefined | null { return super.Section.get<number | null>("maxMemory"); }
+    public get maxConcurrentThreads(): number | undefined | null { return super.Section.get<number | null>("maxConcurrentThreads"); } public get maxMemory(): number | undefined | null { return super.Section.get<number | null>("maxMemory"); }
```

Comment:
Not sure what happened on this line, but can you split it and put the `maxMemory` getter on its own line?